cellmlmanip/_config.py

```diff
@@ -0,0 +1,40 @@
+"""
+Constants and version information.
+"""
+
+import inspect
+#
+# Get package location, find template directory
+#
+import os  # noqa
+
+
+try:
+    frame = inspect.currentframe()
+    MODULE_DIR = os.path.dirname(inspect.getfile(frame))
+finally:
+    # Always manually delete frame
+    # https://docs.python.org/2/library/inspect.html#the-interpreter-stack
+    del(frame)
+
+#
+# Version info
+#
+with open(os.path.join(MODULE_DIR, 'version.txt'), 'r') as f:
```

Comment:
Is this the recommended way of approaching this problem? Or would [pkg_resources](https://setuptools.readthedocs.io/en/latest/pkg_resources.html#basic-resource-access) be better?

Reply:
it's one of the recommendations on here:
https://packaging.python.org/guides/single-sourcing-package-version/
Since the docs conf.py doesn't (directly) import cellmlmanip, pkg_resources  probably wouldn't work but the version.txt approach should